source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -283,35 +283,56 @@ commands as a shell script to configure all of these services in one step:
 Troubleshooting
 ***************
 
-* <HOSTNAME> not found in <IP> message. 
+* <HOSTNAME> not found in <IP> message.
+
+  Your DNS server may not be appropriately configured. Try adding an
+  entry to the :file:`/etc/hosts` file with your host's IP and Name.
+  
+  For example: 100.200.50.20 myhost
 
-  Your DNS server may not be appropriately configured. You can try adding
-  an entry to the :file:`/etc/hosts` file with your host's IP and Name. Use
-  the commands *hostname* and *hostname -I* to retrieve them.
-  For example: 100.200.50.20   myhost
+  Use the commands :command:`hostname` and :command:`hostname -I` to retrieve them.
 
-* Images cannot be pulled. 
+* Images cannot be pulled.
 
-  You may be behind a proxy server. Try configuring your proxy settings, 
+  You may be behind a proxy server. Try configuring your proxy settings,
   using the environment variables *HTTP_PROXY*, *HTTPS_PROXY*, and *NO_PROXY*
   as required in your environment.
 
-* Connection refused error. 
+* Connection refused error.
 
   If you are behind a proxy server, you may need to add the master's IP to
   the environment variable *NO_PROXY*.
 
 * Connection timed-out or Access Refused errors.
 
-  You must ensure that the appropriate proxy settings are available from the 
-  same terminal where you will initialize the control plane. To verify the 
-  proxy settings that kubernetes will actually use, run the commands:
+  You must ensure that the appropriate proxy settings are available from the
+  same terminal where you will initialize the control plane. To verify the
+  proxy settings that Kubernetes will actually use, run the commands:
+
+  .. code-block:: bash
+
+    echo $HTTP_PROXY
+    echo $HTTPS_PROXY
+    echo $NO_PROXY
+
+  If the displayed proxy values are different from your assigned values, the
+  cluster initialization will fail. Contact your IT support team to learn how
+  to set the proxy variables permanently, and how to make them available for
+  all the types of access that you will use, such as remote SSH access.
 
-  * *"echo $HTTP_PROXY"* 
-  * *"echo $HTTPS_PROXY"* 
-  * *"echo $NO_PROXY"* 
+* Missing environment variables.
```

Comment:
@bktan8  Please confirm or update the error message in line 323.

Reply:
LGTM